UnityProject/Assets/Scripts/UI/Core/Action/UIAction.cs

```diff
@@ -100,12 +105,57 @@ public void ButtonPress()
 			if (actionData.IsToggle)
 			{
 				Toggle();
+				if (!actionData.IsAimable)
```

Comment:
As good practice, we tend to avoid using !bool. Use bool == false or bool == true instead, it is better for readability, t can be quite easy to miss the ! sometimes